main.go

```diff
@@ -323,19 +322,15 @@ func main() {
 		configs.ApkFileIncludeFilter = "*.apk"
 	}
 
+	if !strings.HasPrefix(configs.GradlewPath, ".") && !strings.HasPrefix(configs.GradlewPath, "/") {
```

Comment:
if !filepath.IsAbs(configs.GradlewPath) && !strings.HasPrefix(configs.GradlewPath, "./")

Reply:
or just expand the path in the main.go and no need to modify the path